### DISCUSSION TITLE
How to get the dist JS file back?

### DISCUSSION THREAD

**Discussion by Guite** [Leaflet/Leaflet#8491] [Q&A] [answered]
Hello,

is there a way to get `leaflet.js` and `leaflet.min.js` for version 1.9.x?

I saw there is something [at unpkg.com](https://unpkg.com/leaflet@1.9.1/dist/leaflet.js) but the release assets / downloads do not contain JS files within the `dist` folder.

Thanks in advance for your assistance.

**Comment by IvanSanchez** [comment]
`leaflet.min.js`? We don't use that since.... a long, long time ago. I mean, a minimum of 7 years.

See https://github.com/Leaflet/Leaflet/pull/3595/files - that should make things clear for you.

**Reply by Guite** [reply]
It was still contained in `leaflet.zip` of release 1.8.0 though (see assets at https://github.com/Leaflet/Leaflet/releases/tag/v1.8.0 for the download). Hence, the change was a bit unexpected in 1.9.0 :-)

**Reply by Falke-Design** [reply]
`leaflet.min.js` was not in the `leaflet.zip`
![image](https://user-images.githubusercontent.com/19800037/192979081-916ee213-db68-4387-ae5b-e16efac44083.png)

But I think you mean, that the complete `dist` folder is not attached to the Release 1.9.X @mourner @jonkoops 


Release 1.8.0:
![image](https://user-images.githubusercontent.com/19800037/192979625-b9a2aefe-1abb-485c-abf9-fc5587dba31d.png)

Release 1.9.0:
![image](https://user-images.githubusercontent.com/19800037/192979720-13e9321e-9b60-4c08-b526-47ad04859791.png)

**Reply by IvanSanchez** [reply]
I've written down issue #8492 to keep track of this (discussions are not bug reports)

**Comment by jonkoops** [comment, marked as answer]
You can download a ZIP archive containing the files from the distribution from the [downloads page](https://leafletjs.com/download.html) on the website.
